CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ description: |-
 - Fix panic on VKCS Kubernetes API error in vkcs_kubernetes_clustertemplates data source
 - Deprecate ingress_floating_ip field of vkcs_kubernetes_cluster resource and data source
 - Deprecate keypair field of vkcs_kubernetes_cluster resource and data source
+- Refactoring of data source identifiers
```

Comment:
Maybe so?
- Add the ability to use the "id" field instead of "*_id" in data sources.
- Deprecate '*_id' field of data sources.

Reply:
Looks better, a couple of enhancements:
* Add the ability to specify the "id" field instead of "<resource>_id" in data sources.
* Deprecate '<resource>_id' field of data sources.
There are fields of the form <*_id> that have different meaning.

Reply:
Should escape angle brackets, as they are Markdown special symbols, don't you?

Reply:
Fixed it, forgot that it was needed